src/main/java/frc/robot/RobotContainer.java

```diff
@@ -186,6 +189,7 @@ public RobotContainer(RobotBase robot) {
       configureButtonBindings();
       configureObjects();
       buildAutoEventMap();
+      ledSubsystem.setAllianceColor();
```

Comment:
Why call this in the RobotContainer constructor instead of the ledSubsystem constructor?